pkg/cloudprovider/vsphere/nodemanager.go

```diff
@@ -138,6 +139,25 @@ func (nm *NodeManager) shakeOutNodeIDLookup(ctx context.Context, nodeID string,
 	return nil, err
 }
 
+func returnIPsFromSpecificFamily(family string, ips []string) []string {
+	var matching []string
+
+	for _, ip := range ips {
+		if ErrOnLocalOnlyIPAddr(ip) != nil {
+			klog.V(4).Info("IP is Local Only")
```

Comment:
This checks invalid IPs as well, probably worth logging what the errror here is.

Reply:
Yup, true.